### PR TITLE
feat(OffscreenCanvas): Support WebWorkers

### PR DESCRIPTION
**Pull request by 2xAA**
Great library, thanks!

This PR adds a private function to replace `document.createElement('canvas')` to allow the return of OffscreenCanvas when the library is used in a WebWorker.

Of course this only works in browsers with OffscreenCanvas support, but doesn't break compatibility with main thread usage and browsers without OffscreenCanvas.

Let me know if you want a WebWorker test added - can stall this PR until that has been done.

### CODE REVIEW

**Comment by 2xAA** [comment]
The Travis build passed, but didn't update on GitHub 🤔

File: src/index.js [hunk 1]
```diff
@@ -405,4 +405,12 @@ export default class FastAverageColor {
 
         return result < 128;
     }
+
+    _makeCanvas() {
+        if (typeof window !== 'undefined') {
+            return document.createElement('canvas');
+        }
+
+        return new OffscreenCanvas(256, 256);
```
[the remark this answers is not in the document]
Comment (2xAA) [reply src/index.js]: Ah, yes okay. 256 was for WebGL textures in another project I'm using OffscreenCanvas with.

I think it can be 1.

**Comment by hcodes** [comment]
#19 
Thanks for your help.

**Comment by hcodes** [comment]
v4.3.0

**Pull request closed without merging**